src/lib/gundb/UserStorageClass.js

```diff
@@ -280,7 +294,7 @@ export const getReceiveDataFromReceipt = (receipt: any) => {
     logs.filter(log => {
       return log && log.name === CONTRACT_EVENT_TYPE_TRANSFER
     }),
-    'value'
+    log => log.value // "value" won't work for unknown reasons probably some lodash's 4.17.15 bug
```

Comment:
this is because of lodash webpack
need to enable 'shorthands'

Reply:
Understand 
Yep, I saw a task about it